game/scripts/npc/abilities/obsidian_destroyer_arcane_orb_oaa.txt

```diff
@@ -25,7 +25,7 @@
 
     // Cost
     //-------------------------------------------------------------------------------------------------------------
-    "AbilityManaCost"                                     "110 140 170 200 400 600"
+    "AbilityManaCost"                                     "100 145 190 235 500 750"
```

Comment:
I think I told you increasing mana cost have no effect on OD when he is farmed because of manacost reduction items. You are just nerfing OD's early-mid game when he doesn't have those items. You probably think that with the other change (max mana increase) this is fine. Max mana is increased only when hitting heroes in case you don't know.

Reply:
I actually tested OD out myself and reflected this change based on the current mana costs without mana reducing items. Effectively this change makes it so that the mana cost we initially had it as is what it will be post mana reduction items. I hope you can understand this nonsense  lol